.gitlab-ci.yml

```diff
@@ -483,6 +493,75 @@ build-rust-doc-release:
     - sccache -s
 
 #### stage:                        post-build-test
+check-polkadot-companion-status:
```

Comment:
I am a bit confused. How does this pull request relate to the polkadot-companion-check?

Reply:
This is generally enabled for all PR atm. @TriplEight Hi Dennis, can you confirm with this Job?

Reply:
Now you have this job [doubled](https://github.com/paritytech/substrate/pull/5793/files#diff-96edf7a6f008de9e928d04e1ae5e12a5R382). It should be a YAML syntax error.
Also do not get the point how is this job related to this PR.

Reply:
Removed.